terraform/modules/commit_review_status/cloud_run_job.tf

```diff
@@ -76,3 +77,38 @@ resource "google_project_iam_member" "default" {
   member = google_service_account.default.member
   role   = "roles/secretmanager.secretAccessor"
 }
+
+// Give the scheduler invoker permission to the cloud run instance
+resource "google_cloud_run_service_iam_member" "invoker" {
+  project = var.project_id
+
+  location = var.region
+  service  = google_cloud_run_v2_job.default.name
+  role     = "roles/run.invoker"
+  member   = google_service_account.default.member
+
+  depends_on = [
+    google_cloud_scheduler_job.scheduler
+  ]
+}
+
+resource "google_cloud_scheduler_job" "scheduler" {
+  project = var.project_id
+
+  name             = "artifacts-job"
```

Comment:
I assume this is a copy paste error?

Reply:
yes, I actually meant to use the `var.job_name` instead of hardcoding it. updated now.